### PR TITLE
[IMP] Helpdesk: customer ratings

### PR DESCRIPTION
**Pull request by jero-odoo**
Updating Helpdesk Customer ratings doc for V17 (content and screenshots)

### CODE REVIEW

**Comment by jero-odoo** [comment]
Hi @tiku-odoo this is ready for review. Thanks!

File: content/applications/services/helpdesk/overview/ratings.rst [hunk 1]
```diff
@@ -3,15 +3,17 @@ Customer ratings
 ================
 
 Asking customers to rate the support they received from a *Helpdesk* team provides an opportunity to
-gauge team performance and track customer satisfaction. Ratings can be published to the portal,
+gauge team performance and track customer satisfaction. Ratings can be published on the portal,
 providing customers with a general overview of the team's performance.
 
+.. _helpdesk/enable-ratings:
+
 Enable customer ratings on a Helpdesk team
```
[the remark this answers is not in the document]
Comment (tiku-odoo) [reply content/applications/services/helpdesk/overview/ratings.rst:11]: Fix titles throughout remove "a" or "an" or "the"

File: content/applications/services/helpdesk/overview/ratings.rst [hunk 2]
```diff
@@ -24,21 +26,21 @@ Set a ratings request email template on a stage
...
+from the list, or click :guilabel:`New` to create a new stage.
 
 .. important::
    Customers should only be asked to rate tickets once an issue has been resolved and their ticket
-   is closed. Therefore, a *ratings request* email should only be added to a stage that is
-   **folded** in the kanban, as tickets in a *folded stage* are considered closed.
+   is *closed*. Therefore, a *ratings request* email should **only** be added to a stage that is
+   folded in the Kanban, as tickets in a *folded stage* are considered closed.
 
 On the stage's settings page, select `Helpdesk: Ticket Rating Request` in the :guilabel:`Email
```
Comment (tiku-odoo) [inline content/applications/services/helpdesk/overview/ratings.rst:38]:
```suggestion
On the stage's settings page, select the template, `Helpdesk: Ticket Rating Request` in the :guilabel:`Email
```

File: content/applications/services/helpdesk/overview/ratings.rst [hunk 3]
```diff
@@ -24,21 +26,21 @@ Set a ratings request email template on a stage
...
-Template` field. This template has been pre-configured with ratings customers can use to provide
+Template` field. This template has been preconfigured with ratings customers can use to provide
 feedback. To view the template, click the arrow button to the right of the field.
```
Comment (tiku-odoo) [inline content/applications/services/helpdesk/overview/ratings.rst:40]:
```suggestion
feedback. To view the template, click the arrow button to the right of the field.
```

Maybe good to use icon here? (arrow)

File: content/applications/services/helpdesk/overview/ratings.rst [hunk 4]
```diff
@@ -24,21 +26,21 @@ Set a ratings request email template on a stage
...
 
-Once the template has been added to the stage, it will automatically send a message when a ticket is
-moved to that stage. Customers will be asked to rate the support they received with colored icons.
+After the template is added to the stage, it automatically sends a message when a ticket is moved to
+that stage. Customers are then asked to rate the support they received with colored icons.
```
Comment (tiku-odoo) [inline content/applications/services/helpdesk/overview/ratings.rst:43]:
```suggestion
that stage. Customers are then asked to rate the support they received with colored icons.
```

How are they asked? A popup? Email sent?

File: content/applications/services/helpdesk/overview/ratings.rst [hunk 5]
```diff
@@ -68,13 +70,15 @@ team has received over the last thirty days. Specific written feedback will not
 statistics of the team's performance will be visible.
 
 .. important::
-   In order to display ratings on the customer portal, a team has to have their visibility setting
-   set to :guilabel:`Invited portal users and all internal users`. This setting is found on the
-   team's settings page under :guilabel:`Visibility`.
+   To display ratings on the customer portal, a team **must** have their visibility setting set to
+   :guilabel:`Invited portal users and all internal users`. To enable this setting, navigate to
+   :menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams`. Select a team from the list
+   and navigate to the settings page. Scroll to the :guilabel:`Visibility & Assignment` section, and
```
Comment (tiku-odoo) [inline content/applications/services/helpdesk/overview/ratings.rst:76]:
```suggestion
   and navigate to the settings page. Scroll to the :guilabel:`Visibility & Assignment` section, and
```

Doesn't it land on the settings page? if not how do I get there?

File: content/applications/services/helpdesk/overview/ratings.rst [hunk 6]
```diff
@@ -49,12 +51,12 @@ moved to that stage. Customers will be asked to rate the support they received w
...
+feedback to support their rating. The rating is then submitted, and the rating, as well as any
+additional comments, are added to the chatter on the ticket.
 
 .. tip::
    Customer ratings can also be viewed through the :guilabel:`Customer Ratings` report. To view
-   this report, go to :menuselection:`Helpdesk --> Reporting --> Customer Ratings`.
+   this report, go to :menuselection:`Helpdesk app --> Reporting --> Customer Ratings`.
 
 .. seealso::
    :doc:`/applications/general/companies/email_template`
```
Comment (tiku-odoo) [inline content/applications/services/helpdesk/overview/ratings.rst:62]: After enabling the Customer Ratings setting, an option to publish ratings on the team’s website appears. Enabling this setting provides portal users with an overview of the ratings the team has received over the last thirty days. Specific written feedback will not be included; only statistics of the team’s performance will be visible.


Where does the option appear?

File: content/applications/services/helpdesk/overview/ratings.rst [hunk 7]
```diff
@@ -68,13 +70,15 @@ team has received over the last thirty days. Specific written feedback will not
...
 
 .. important::
-   In order to display ratings on the customer portal, a team has to have their visibility setting
-   set to :guilabel:`Invited portal users and all internal users`. This setting is found on the
-   team's settings page under :guilabel:`Visibility`.
+   To display ratings on the customer portal, a team **must** have their visibility setting set to
+   :guilabel:`Invited portal users and all internal users`. To enable this setting, navigate to
+   :menuselection:`Helpdesk app --> Configuration --> Helpdesk Teams`. Select a team from the list
+   and navigate to the settings page. Scroll to the :guilabel:`Visibility & Assignment` section, and
+   tick the :guilabel:`Invited portal users and all internal users` checkbox.
```
Comment (tiku-odoo) [inline content/applications/services/helpdesk/overview/ratings.rst:77]:
```suggestion
   tick the :guilabel:`Invited portal users and all internal users` checkbox.
```

this isn't the exact thing on the UI

Should be: Invited portal users and all internal users (public)

File: content/applications/services/helpdesk/overview/ratings.rst [hunk 8]
```diff
@@ -68,13 +70,15 @@ team has received over the last thirty days. Specific written feedback will not
...
 
-Next, to publish the ratings, go to :menuselection:`Helpdesk --> Configuration --> Teams` and
-select a team. Scroll to :guilabel:`Performance` and enable :guilabel:`Publish this team's ratings
-on your website`.
+Next, to publish the ratings, go to :menuselection:`Helpdesk app--> Configuration --> Helpdesk
+Teams` and select a team. Scroll to :guilabel:`Performance` and enable :guilabel:`Publish this
```
Comment (tiku-odoo) [inline content/applications/services/helpdesk/overview/ratings.rst:80]:
```suggestion
Teams` and select a team. Scroll to :guilabel:`Performance` and tick the checkbox for :guilabel:`Publish this
```

File: content/applications/services/helpdesk/overview/ratings.rst [hunk 9]
```diff
@@ -88,14 +92,14 @@ Manually hide individual ratings
...
 
 To make a rating visible only to internal users, navigate to the page for a rating. This can be done
 in one of the following ways:
 
-   - Go to :menuselection:`Helpdesk --> Reporting --> Customer Ratings` and click on one of the
-     kanban cards for an individual rating.
-   - Navigate to :menuselection:`Helpdesk --> Tickets --> All Tickets` and remove the
+   - Go to :menuselection:`Helpdesk app --> Reporting --> Customer Ratings` and click on one of the
+     Kanban cards for an individual rating.
+   - Navigate to :menuselection:`Helpdesk app--> Tickets --> All Tickets` and remove the
```
Comment (tiku-odoo) [inline content/applications/services/helpdesk/overview/ratings.rst:102]: Maybe add another see also to the portal documentation

https://www.odoo.com/documentation/17.0/applications/general/users/portal.html

Comment (tiku-odoo) [inline content/applications/services/helpdesk/overview/ratings.rst:102]: Once on the rating details page, tick the Visible Internally Only box.

**Review by tiku-odoo — APPROVED** [review]
@jero-odoo 

Great work on this doc! I made a few suggestions for you to take into consideration. 

Approving and ready to move onto the next stage. 

Thanks :+1: 

Tim

**Comment by jero-odoo** [comment]
hey @samueljlieber this is ready for final/tech review. Thanks!

**Review by samueljlieber — APPROVED** [review]
Hi @jero-odoo, nice quick updates to this doc. Regarding the images– there doesnt appear to be any change in the versions (besides the company name) in what is shown in the new images introduced in this PR. I don't see a need to update with these images as doing so takes more of the server's resources. 

Approving and delegating to you, please consider removing images that dont _need_ to be updated :)
..
@robodoo delegate=jero-odoo

**Comment by jero-odoo** [comment]
@robodoo r+

**Pull request closed without merging**